doc/group_instances.md

```diff
@@ -0,0 +1,269 @@
+### Group Instance & Group Instance IDs
+
+---
+
+It is recommended that you familiarise yourself with [*repeating routing rules*](repeating_routing_rules.md) before proceeding with this.
+
+Group Instance / Instance IDs are used to identify the instance of an answer acquired from a repeating group.
+
+#### Group Instance (Index)
+An answer can have multiple instances, the *group_instance* tracks this instance and is part of the [*Location*](../app/questionnaire/location.py) object which consists of *group_id*, *group_instance*, and the *block_id*.
```

Comment:
I didn't talk about answer_instance as we no longer use them. `RepeatingAnswer` is only in the test schemas and currently census but I assume it's just not updated.

Do you think it would be good to define answer_instance and how they work?

Reply:
I think it would be worth acknowledging the existence of answer instances and just say that they are deprecated.